dotcom-rendering/src/components/AudioAtom/AudioAtom.tsx

```diff
@@ -98,10 +93,14 @@ const progressBarStyle = css`
 	display: block;
 `;
 
-const progressBarInputStyle = (color: string) => css`
+const progressBarInputStyle = css`
 	width: 100%;
 	appearance: none;
-	background-image: linear-gradient(to right, ${color} 0%, ${neutral[60]} 0%);
+	background-image: linear-gradient(
+		to right,
+		${palette('--audio-atom-icons')} 0%,
+		${palette('--audio-atom-progress-bar')} 0%
```

Comment:
What does this achieve?

Reply:
It draws a progress bar? Sorry, I'm not sure I understand the question! 😬

Reply:
I'm just having a hard time parsing the CSS—but if it works visually, it's fine!